include/wolfboot/wolfboot.h

```diff
@@ -136,17 +138,17 @@ extern "C" {
 /* ML-DSA pub key size is a function of parameters.
  * This needs to be configurable. Default to security
  * category 2. */
```

Comment:
If we're defaulting to ML_DSA_LEVEL=5 then update or delete this comment please!